### DISCUSSION TITLE
How to find sponsors for my unpopular project

### DISCUSSION THREAD

**Discussion by vishalkrishna8** [community/community#69179] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I am looking for a sponsor for my project, can anyone give me some suggestions or advices on how to find a sponsor as quickly as possible.

**Comment by cavalluccijohann** [comment]
Hey!

You can simply use GitHub Sponsors to find support for your project. Just talk about your project, and if someone is interested in supporting you, they can do so directly through GitHub. I'm not sure if this helps, but I hope it does!

☀️
